spec/system/candidate_entering_qualification_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe 'A candidate adding a qualification' do
+  include TestHelpers::NonDegreeQualifications
+
+  context 'who successfully enters their details' do
+    before do
+      visit '/qualifications/new'
+
+      fill_in_qualification_details
+
+      click_on t('application_form.save_and_continue')
```

Comment:
have you folks thought about using page objects? The Find team's had good experiences with [SitePrism](https://github.com/natritmeyer/site_prism).

Reply:
hey @benilovj, thanks for the suggestion, SitePrism was new for me, I've just read it's docs. 
It looks like it is good to use when 

> ... when it is necessary to perform some kind of ugly voodoo with xpaths that cover multiple levels of nodes.
> (from https://www.pgs-soft.com/blog/better-automated-tests-in-ruby-using-the-siteprism-gem-for-the-page-object-model-pattern/)

At the moment, the methods provided by Cabybara are enough to cover the basic find/fill/click_on cases, but it is a good tool that we'll probably use when things will become complex.

Reply:
@benilovj we've thought about it a bit (back when it was just @duncanjbrown and I), and settled on using the simple modules for the particularly gnarly bits of the form to start. 

We might evolve towards something more sophisticated as we grow the app, thanks for the tool suggestions :+1: